.github/workflows/build.yaml

```diff
@@ -139,6 +139,49 @@ jobs:
         if: steps.changed-files.outputs.any_changed == 'true'
         run: yarn lint:ts
 
+  test-unit:
```

Comment:
Since we no longer need a build to run unit tests, it can now run in parallel with our other pre-build jobs 🎉

Reply:
:tada: :tada: :tada: